models/Device.ts

```diff
@@ -45,8 +45,10 @@ export interface Device extends Sequelize.Model, ModelCommon<Device> {
     group: Group,
     newPassword: string
   ) => Promise<Device>;
-
+  Group: Group;
```

Comment:
Doesn't make sense to have group and group ID?

Reply:
Why? groupId belongs to Device table, if you fetch a Device with the associated Group it will be under the name Group. If you don't define it TS  throws a fit